app/src/main/java/umc/everyones/lck/presentation/home/HomeFragment.kt

```diff
@@ -1,17 +1,46 @@
 package umc.everyones.lck.presentation.home
 
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
 import dagger.hilt.android.AndroidEntryPoint
 import umc.everyones.lck.R
 import umc.everyones.lck.databinding.FragmentHomeBinding
 import umc.everyones.lck.presentation.base.BaseFragment
+import umc.everyones.lck.presentation.login.LoginActivity
+import umc.everyones.lck.presentation.mypage.MyPageActivity
+import umc.everyones.lck.util.LoginManager
 
 @AndroidEntryPoint
 class HomeFragment : BaseFragment<FragmentHomeBinding>(R.layout.fragment_home) {
-    override fun initObserver() {
 
+    override fun initObserver() {
+        setupLogoutButton()
+        setupMypageButton()
     }
 
     override fun initView() {
+        // Initialize any view-related properties here if needed
+    }
```

Comment:
```suggestion
    override fun initObserver() {
       
    }

    override fun initView() {
        setupLogoutButton()
        setupMypageButton()
    }
```
initView로 함수 호출 옮겨주세요